ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/ui/vr/RDXVRKinematicsStreamingMode.java

```diff
@@ -322,15 +323,13 @@ public void processVRInput()
          InputDigitalActionData aButton = controller.getAButtonActionData();
          if (aButton.bChanged() && !aButton.bState())
          {
-            streamToController.set(!streamToController.get());
-            if (!streamToController.get())
-               streamingDisabled.set();
+            setStreamingEnabled(!streamingEnabled.get());
          }
 
          InputDigitalActionData bButton = controller.getBButtonActionData();
          if (bButton.bChanged() && !bButton.bState())
          {
-            ghostPreviewEnabled.set(!ghostPreviewEnabled.get());
+            showGhostsWhileStreaming.set(!showGhostsWhileStreaming.get());
```

Comment:
This toggle will persist and is also bound to the checkbox. I don't want it to auto reset. The ghosts always show in preview mode.